File: Java/Cuenta.java
package Java;

/**
 * Cuenta
 */
public class Cuenta {

    //Atributos del objeto Cuenta
    Integer id;
    String name;
    String document;
    String email;
    String password;

    //método constructor
    public  Cuenta(String name, String document) {
        this.name = name;
        this.document = document;
        
    }

}

File: Python/cuenta.py
class Cuenta:
    # Astributos de las clase Cuenta
    id          = int
    name        = str
    document    = str
    email       = str
    password    = str

    # Método constructor
    def __init__(self, name, document):
        self.name     = name
        self.document = document